src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2972,22 +2975,22 @@ private static Stream ConditionalGetOrCreateStream(EmitStreamProvider metadataPE
             }
         }
 
-        internal string Feature(string p)
+        internal string? Feature(string p)
         {
             string v;
             return _features.TryGetValue(p, out v) ? v : null;
         }
 
         #endregion
 
-        private ConcurrentDictionary<SyntaxTree, SmallConcurrentSetOfInts> _lazyTreeToUsedImportDirectivesMap;
+        private ConcurrentDictionary<SyntaxTree, SmallConcurrentSetOfInts>? _lazyTreeToUsedImportDirectivesMap;
         private static readonly Func<SyntaxTree, SmallConcurrentSetOfInts> s_createSetCallback = t => new SmallConcurrentSetOfInts();
 
         private ConcurrentDictionary<SyntaxTree, SmallConcurrentSetOfInts> TreeToUsedImportDirectivesMap
         {
             get
             {
-                return LazyInitializer.EnsureInitialized(ref _lazyTreeToUsedImportDirectivesMap);
+                return LazyInitializer.EnsureInitialized(ref _lazyTreeToUsedImportDirectivesMap)!;
```

Comment:
📝 This is an annotations gap for not targeting netcoreapp3.0. Consider adding `RoslynLazyInitializer` so we can make this call without suppressions.

Reply:
Good idea. Will do that. 

---
In reply to: [359960511](https://github.com/dotnet/roslyn/pull/40493#discussion_r359960511) [](ancestors = 359960511)